classes/pricing/aws_base_pricing_client.php

```diff
@@ -77,7 +63,7 @@ class aws_ets_pricing_client {
     /**
      * The ServiceCode for Amazon Elastic Transcode Services.
```

Comment:
fix comment

Reply:
there is a bunch below too outside the diff which github won't let me comment on, just general grep for Elastic Transcode Services and ETS